.github/CONTRIBUTING.md

```diff
@@ -4,6 +4,15 @@ This doc has some guidelines on how to generate and contribute to this library.
 
 ## Generate
 
+
+export const toMessagePublishedEvent = (json: object) => {
+    return json as MessagePublishedEvent;
+};
+
```

Comment:
Do we need all these blank lines?

Reply:
Not sure which blank lines we're talking about.

The paragraphs/lists are separated by 1 blank line (rather than 0 blank lines). Doesn't look different than other libraries.

Reply:
I think I was probably trying to review one commit at a time, and it was resolved later by removing the extra lines.